proxy_network.c

```diff
@@ -1334,6 +1333,7 @@ static int _flush_pending_write(mcp_backend_t *be) {
         }
         // still have unflushed pending IO's, check for write and re-loop.
         if (be->io_next) {
+            be->can_write = false;
```

Comment:
For my understanding, what do you mean by short writes? Do you mean the writes that still have pending IO left?

Reply:
Any write syscall can return fewer bytes sent than you attempted to write. I call those short writes.